### ISSUE TITLE
Long text should wrap around instead of resizing to smaller font size

### ISSUE THREAD

**Issue by emilychang9** [Hap-Map/HapMap#12]
When a long address is displayed on the confirmation page, the text is resized to a small font instead of wrapping around and resizing the card as needed. This can cause accessibility issues for those who cannot read small text or are unable to turn the device to landscape mode.

![Screenshot_1651214086](https://user-images.githubusercontent.com/69648669/165895700-3b7fbf2f-d0f7-49bd-a1ea-4679d71c4a91.png)
